pkg/services/accesscontrol/resourcepermissions/service.go

```diff
@@ -23,7 +26,7 @@ import (
 	"github.com/grafana/grafana/pkg/setting"
 )
 
-var _ pluginaccesscontrol.ActionSetRegistry = (*InMemoryActionSets)(nil)
+var _ pluginaccesscontrol.ActionSetRegistry = (ActionSetService)(nil)
```

Comment:
yea i like this refactoring. the naming makes sense and just separating the svc vs the store.